src/ZendeskApi.Client/Resources/RequestResource.cs

```diff
@@ -28,7 +28,7 @@ public IResponse<Request> Get(long id)
 
         public IResponse<Request> Get(IEnumerable<TicketStatus> requestedStatuses)
         {
-            return GetAsync(requestedStatuses).Result;
+            return GetAsync(requestedStatuses).ConfigureAwait(false).GetAwaiter().GetResult();
```

Comment:
@EdNet I've reworked all of the synchronous calls to not just call awaiters on async methods any more. See https://github.com/justeat/ZendeskApiClient/pull/57 for more info but BCL is having some issues so I can't get it out